docs/source/federation/core-concepts.md

```diff
@@ -164,3 +164,49 @@ extend type Query {
 ```
 
 There is no need to explictly define `Query` or `Mutation` base types anywhere; Apollo automatically handles this for you.
+
+## Sharing Types
+
+Some types need to be shared across services. Specifically, it's very useful to share Enum and Scalar types across all services to prevent code duplication.
```

Comment:
This isn't really about preventing duplication (it causes it in fact!) but is more about final schema consistency